### DISCUSSION TITLE
How to change dDragon version?

### DISCUSSION THREAD

**Discussion by JakubAmbroziak** [TheDrone7/shieldbow#38] [Q&A] [answered]
Hi, can u tell me, how can i change the version of dDragon API uses? I believe Riot released patch today and that's why dDragon is working correctly, but is there any way to set dDragon's version to prepatch?

**Comment by TheDrone7** [comment, marked as answer]
The version defaults to the latest as per the version files from ddragon, but you can always manually set the version by specifying the version property in the initialisation options, please refer to the [client initialisation documentation](https://thedrone7.github.io/shieldbow/guide/client.html#client-initialization) for details.
